### ISSUE TITLE
Unresolved dependency found during sorting - net-ssh (>= 2.6.5) (requested by net-scp-1.2.1) since 2.2.11

### ISSUE THREAD

**Issue by micheelengronne** [hashicorp/vagrant#12111]
### Vagrant version

Since 2.2.11

### Host operating system

Ubuntu groovy (20.10).

Note that groovy only support ruby 2.7

### Expected behavior

Vagrant should install properly

### Actual behavior

Vagrant does not install and display this error:

```sh
ERROR:  While executing gem ... (Gem::DependencyError)
    Unresolved dependency found during sorting - net-ssh (>= 2.6.5) (requested by net-scp-1.2.1)
```

### Steps to reproduce

1. git clone https://github.com/hashicorp/vagrant.git
2. cd vagrant
3. git checkout 2.2.11   # 2.2.12, 2.2.13 and 2.2.14
4. gem build vagrant.gemspec
5. gem install vagrant-2.2.11.gem

### References

- https://github.com/hashicorp/vagrant/issues/12022

**Comment by soapy1** [comment]
Hi there,

It looks like this has been resolved within a previously shipped version of Vagrant so I am now closing this issue. If the original issue was not fully resolved, please reopen this issue or create a new one.

Cheers!

**Comment by micheelengronne** [comment]
Hi @soapy1 it is not solved in the last release (2.2.14)

Please, can you reopen ?

**Comment by soapy1** [comment]
ah, yep I'm mistaken. This error is being caused by the [`net-ssh` dependency](https://github.com/hashicorp/vagrant/blob/main/vagrant.gemspec#L27). It can be resolved once [net-ssh 6.2.0](https://github.com/net-ssh/net-ssh/releases) is released.

**Comment by chrisroberts** [comment]
fixed via #12213 and will be included in the next release
